src/libraries/Microsoft.PowerFx.Core/Public/DisplayNames/SingleSourceDisplayNameProvider.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace Microsoft.PowerFx.Core
 {
-    internal class SingleSourceDisplayNameProvider : DisplayNameProvider
+    public class SingleSourceDisplayNameProvider : DisplayNameProvider
```

Comment:
nit: can we keep the class private, and instead have a static builder on the base DisplayNameProvider class?

Reply:
good point, updated.